src/assets/tailwind.css

```diff
@@ -4,51 +4,108 @@
 
 @tailwind utilities;
 
+@layer base {
+	label {
+		@apply text-sm font-medium;
+	}
+	input {
+		@apply mb-3 px-2 py-1.5 mb-3 mt-1 block w-full px-2 py-1.5 border border-gray-300 rounded-md text-sm shadow-sm placeholder-gray-400
+    bg-background-secondary
+    focus:outline-none
+    focus:border-sky-500
+    focus:ring-1
+    focus:ring-sky-500
+    focus:invalid:border-red-500 focus:invalid:ring-red-500 text-textcolor-input;
+	}
+	button {
+		@apply px-4 py-1.5 rounded-md shadow-lg bg-sky-800 font-medium text-gray-100 block hover:bg-sky-700 transition duration-300 disabled:bg-background-secondary;
+	}
+	form {
+		@apply flex flex-col justify-center;
+	}
+	textarea {
+		@apply mb-3 px-2 py-1.5 mb-3 mt-1 block w-full px-2 py-1.5 border border-gray-300 rounded-md text-sm shadow-sm placeholder-gray-400
```

Comment:
double px, py (row 12) 
double px, py, mb (row 27 & 36)
dont know if this will cause problem with tailwind yet!